drift-codec/src/main/java/com/facebook/drift/codec/metadata/ThriftType.java

```diff
@@ -48,6 +49,7 @@
     public static final ThriftType STRING = new ThriftType(ThriftProtocolType.STRING, String.class);
     public static final ThriftType BINARY = new ThriftType(ThriftProtocolType.BINARY, ByteBuffer.class);
     public static final ThriftType VOID = new ThriftType(ThriftProtocolType.STRUCT, void.class);
+    public static final ThriftType URI = new ThriftType(ThriftProtocolType.STRUCT, URI.class);
```

Comment:
Hmm, why is this not a `STRING`?

Reply:
I thought had coded this up as a string. I see some test failures when I use string. Let me investigate that.

Reply:
Added a proper type for URI which is needed for the bytecode generation .

Reply:
resolved now with removing custom types.